infra/openshift-manifests/lib.sh

```diff
@@ -68,6 +68,13 @@ function add_minio_webhook(){
 function patch_knative_serving(){
     # patch knative serving to use http instead of https in the service status url
     oc patch knativeserving -n knative-serving knative-serving -p '{"spec":{"config":{"network":{"default-external-scheme": "http"}}}}' --type=merge
+
+    # patch knative serving to only keep one revision of a service
+    kubectl patch configmap/config-gc \
+        -n knative-serving \
+        --type merge \
+        -p '{"data":{"min-non-active-revisions":"0", "max-non-active-revisions":"0"}}'
+
```

Comment:
Changed Knative GC to remove non-active revisions. It is bad for the demo if we show crashing pods, because of missing K_SINK.

We can change later if we like.